services/basic_types.proto

```diff
@@ -1592,3 +1593,32 @@ message StakingInfo {
         int64 staked_node_id = 6;
     }
 }
+
+message VirtualAddress {
+    /**
+     * The 20-byte EVM address.
+     */
+    bytes address = 1;
+
+    /**
+     * The ethereum transaction nonce associated with this address.
+     */
+    int64 nonce = 2;
+
+    /**
+     * Flag if this address should now be set or is the default address on the account.
+     */
+    bool is_default_address = 3;
```

Comment:
When used together with its container as in `VirtualAddress.isDefaultAddress` it seems redundant to include address twice.

```suggestion
    bool is_default = 3;
```

Reply:
Updated.